src/components/list/list.service.js

```diff
@@ -24,4 +26,29 @@ module.exports.getUsersLists = async (accountId) => {
     const accountObjetcId = mongoose.Types.ObjectId(accountId);
 
     return await listModel.find({ accountId: accountObjetcId }); ;
+};
+
+/**
+ * Add a media item to the specified list.
+ * @param {String} listId Optional list id. If not provided the default list is used instead.
+ * @param {String} accountId Account identifier.
+ * @param {*} mediaItem Media item object
+ */
```

Comment:
Nice joob! A little thing, I noticed you could add the next line:

`@throws {InternalServerError} When there's an unhandled error`